pkg/rpc/client/client.go

```diff
@@ -57,8 +57,8 @@ type cache struct {
 // calculateValidUntilBlockCache stores cached number of validators and
 // cache expiration value in blocks
 type calculateValidUntilBlockCache struct {
-	validatorsCount uint32
-	expiresAt       uint32
+	committeeCount uint32
```

Comment:
But why this change? We still can get the number of validators from `getnextblockvalidators`

Reply:
Can we guarantee that `getnextblockvalidators` never returns an empty array?

Reply:
The request can fail, but the network can't really work without validators, so they always exist.